src/ProjectFileTools/GoToDefinition/GoToDefinitionController.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using EnvDTE;
+using Microsoft.VisualStudio;
+using Microsoft.VisualStudio.OLE.Interop;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+using ProjectFileTools.MSBuild;
+
+namespace ProjectFileTools
+{
+    internal class GotoDefinitionController : IOleCommandTarget
+    {
+        private readonly WorkspaceManager _workspaceManager;
+
+        internal GotoDefinitionController(IWpfTextView textview, WorkspaceManager workspaceManager)
+        {
+            TextView = textview;
+            _workspaceManager = workspaceManager;
+        }
+
+        public IWpfTextView TextView { get; private set; }
+
+        public IOleCommandTarget Next { get; set; }
+        public int QueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[] prgCmds, IntPtr pCmdText)
+        {
+            if (pguidCmdGroup == VSConstants.GUID_VSStandardCommandSet97)
+            {
+                if (cCmds == (uint)VSConstants.VSStd97CmdID.GotoDefn)
+                {
+                    prgCmds[0].cmdf = (uint)OLECMDF.OLECMDF_SUPPORTED;
+                    return VSConstants.S_OK;
+                }
+            }
+
+            return Next.QueryStatus(pguidCmdGroup, cCmds, prgCmds, pCmdText);
```

Comment:
`Next` can be null, use a null operator.  `Next?.QueryStatus(...) ?? Constants.OLECMDERR_E_NOTSUPPORTED`. Also see my comment regarding the setting of `Next`.